extensions/smallrye-openapi/spi/src/main/java/io/quarkus/smallrye/openapi/deployment/spi/OpenAPIBuildItem.java

```diff
@@ -0,0 +1,21 @@
+package io.quarkus.smallrye.openapi.deployment.spi;
+
+import org.eclipse.microprofile.openapi.OASFilter;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+public final class OpenAPIBuildItem extends MultiBuildItem {
```

Comment:
Maybe this should be more specific? Like `OpenAPIFilterBuildItem`?

Reply:
I'm wondering if it needs to be super specific to what we're actually doing.

Though what's being included is an implementation of `OASFilter`, maybe the build item should be something like `IncludeEndpointInOpenAPIDefinitionBuildItem`? I know, very wordy, but elaborates what it's for

Reply:
Also maybe an `OASFilter` is just an implementation detail understood by the OpenAPI extension. I wonder if this build item shouldn't just contain the data (Endpoint path, etc)

Reply:
Something like `AdditionalOpenAPIEndpointBuildItem`

Reply:
Ken made the same comment, I'll go with his very specific suggestion (IncludeEndpointInOpenAPIDefinitionBuildItem)